lib/matplotlib/backends/backend_ps.py

```diff
@@ -374,6 +369,37 @@ def get_image_magnification(self):
         """
         return self.image_magnification
 
+    def _convert_path(self, path, transform, clip=False, simplify=None):
+        if clip:
+            clip = (0.0, 0.0, self.width * 72.0, self.height * 72.0)
+        else:
+            clip = None
+        return _path.convert_to_string(
+            path, transform, clip, simplify, None,
+            6, [b"m", b"l", b"", b"c", b"cl"], True).decode("ascii")
+
+    def _get_clip_cmd(self, gc):
+        clip = []
+        rect = gc.get_clip_rectangle()
+        if rect:
+            clip.append("%s clipbox\n" % _nums_to_str(*rect.size, *rect.p0))
+        path, trf = gc.get_clip_path()
+        if path:
```

Comment:
Same here: Changed from `is not None`. Intentional?

Reply:
per the above.

Reply:
Change as well to `is not None`?

Reply:
sure